pkg/auth/handler/webapp/settings.go

```diff
@@ -26,19 +31,72 @@ func ConfigureSettingsRoute(route httproute.Route) httproute.Route {
 		WithPathPattern("/settings")
 }
 
+type SettingsViewModel struct {
+	Authenticators           []*authenticator.Info
+	RecoveryCodes            []*mfa.RecoveryCode
```

Comment:
I don't think we should load all recovery codes upon entering settings page: viewing recovery code logs a security event (which GitHub does), so it should be loaded only on the page specific to viewing recovery code.